modules/pipeline/aop/plugins/task/autotest_cookie_keep_before/plugin.go

```diff
@@ -87,7 +88,19 @@ func (p *provider) Handle(ctx *aoptypes.TuneContext) error {
 	if config.Header == nil {
 		config.Header = map[string]string{}
 	}
-	config.Header[apitestsv2.HeaderCookie] = setCookie
+	var cookie string
+	for key, value := range config.Header {
+		if strings.ToLower(key) == "cookie" {
```

Comment:
`"cookie"` => `apitestsv2.HeaderCookie` and Modify apitestsv2.HeaderCookie to `cookie`

Reply:
No. `cookie` can be passed in from autotest global configs, and the name can be `cookie`, `Cookie` and `COOKIE`.

Reply:
use strings.equalfold